1_OCaml/TP/TP4.ipynb

```diff
@@ -19,7 +19,9 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        | r::q -&gt; (let inter = e::r in inter::(add e q));;
</pre><p>Ce n'est pas nécessaire d'utiliser inter comme variable intermédiaire. Ça alourdit un peu le code d'ailleurs</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>